README.md

```diff
@@ -0,0 +1,94 @@
+# Table of Contents
+1. [Automation Practice Testing](#automation-practice-testing)
+2. [Technologies used](#technologies-used)
+3. [How to install and run the project](#how-to-install-and-run-the-project)
+4. [Contributor Guidelines](#contributor-guidelines)
+   1. [Code review process](#code-review-process)
+   2. [Branches](#branches)
+   3. [Commit message convention](#commit-message-convention)
+
+## Automation Practice Testing
+This application is aimed to run automated test cases on the
+[My Store](https://automationpractice.com/) website. This project is useful
+because it offers quicker feedback regarding the quality of the website.
```

Comment:
I recommend changing this a little bit. "This project is useful
because it offers quicker feedback regarding the quality of the website."

Maybe something like this "This project is useful
because it offers quicker feedback about the main functionality of the online store, and with that, verify if that functionality met the shopping workflow, sign in workflow, etc. "

Reply:
> I recommend changing this a little bit. "This project is useful because it offers quicker feedback regarding the quality of the website."
> 
> Maybe something like this "This project is useful because it offers quicker feedback about the main functionality of the online store, and with that, verify if that functionality met the shopping workflow, sign in workflow, etc. "

Changes made after your review:
- Changing automation practice testing section.
- Changing the commit message convention section and examples.
- Adding the pull requests section.